src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -837,6 +837,22 @@ type Geometry {
     points: Polyline
 }
 
+scalar GeoJson
+
+type StopGeometries {
+    """Representation of the stop geometries as GeoJSON (https://geojson.org/)"""
+    geoJson: GeoJson,
+
+    """
+    Representation of a stop as a series of polylines.
+
+    Polygons of flex stops are represented as linear rings (lines where the first and last point are the same).
+
+    Proper stops are represented as single point "lines".
+    """
+    polylines: [Geometry]
```

Comment:
is polylines a proper name for this?

Reply:
Good point. What do you think would be better one? `googleEncoded`?

Reply:
Sounds good